cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -148,15 +143,17 @@ func Run(c *cloudcontrollerconfig.CompletedConfig, stopCh <-chan struct{}) error
 	}
 
 	// Start the controller manager HTTP server
+	// unsecuredMux is the handler for these controller *after* authn/authz filters have been applied
+	var unsecuredMux *mux.PathRecorderMux
```

Comment:
this change seems to be unrelated to the refactoring. But even it belongs here, I doubt this is correct: this way the controller debug paths are installed only into the insecure serving mux if it is enabled.

Reply:
sorry, my mistakes. The debug function no need for all controller in CCM, just see in `GCcontroller` in KCM.
will revert at once

Reply:
fixed